test/00-administration/02-libfeature.js

```diff
@@ -0,0 +1,21 @@
+import test from 'ava';
+
+var OpenDKIM = require('../../');
+
+test('test lib feature', t => {
+  try {
+    var opendkim = new OpenDKIM();
+
+    opendkim.lib_feature('DKIM_FEATURE_DIFFHEADERS');
+    opendkim.lib_feature('DKIM_FEATURE_PARSE_TIME');
+    opendkim.lib_feature('DKIM_FEATURE_QUERY_CACHE');
+    opendkim.lib_feature('DKIM_FEATURE_SHA256');
+    opendkim.lib_feature('DKIM_FEATURE_DNSSEC');
+    opendkim.lib_feature('DKIM_FEATURE_OVERSIGN');
```

Comment:
Did you end up validating these outputs?  I know the results would be non-deterministic to test against, as this just tells you what libopendkim was compiled with.  I just want to make sure they tossed back what you expected?

Reply:
hmm, if I wrap those in a `console.log()` I get a series of `undefined` values:
```
    console.log(opendkim.lib_feature('DKIM_FEATURE_DIFFHEADERS'));              
    console.log(opendkim.lib_feature('DKIM_FEATURE_PARSE_TIME'));               
    console.log(opendkim.lib_feature('DKIM_FEATURE_QUERY_CACHE'));              
    console.log(opendkim.lib_feature('DKIM_FEATURE_SHA256'));                   
    console.log(opendkim.lib_feature('DKIM_FEATURE_DNSSEC'));                   
    console.log(opendkim.lib_feature('DKIM_FEATURE_OVERSIGN'));                 
```
```
undefined
undefined
undefined
undefined
undefined
undefined
  ✔ test lib feature

  1 test passed [23:13:17]
```

Reply:
to catch this better, we should make sure all of these checks test for type boolean
```
    t.is(typeof opendkim.lib_feature('DKIM_FEATURE_DIFFHEADERS'), 'boolean');                            
    t.is(typeof opendkim.lib_feature('DKIM_FEATURE_PARSE_TIME'), 'boolean');    
    t.is(typeof opendkim.lib_feature('DKIM_FEATURE_QUERY_CACHE'), 'boolean');   
    t.is(typeof opendkim.lib_feature('DKIM_FEATURE_SHA256'), 'boolean');        
    t.is(typeof opendkim.lib_feature('DKIM_FEATURE_DNSSEC'), 'boolean');        
    t.is(typeof opendkim.lib_feature('DKIM_FEATURE_OVERSIGN'), 'boolean');
```